plugins/vault/src/components/EntityVaultTable/EntityVaultTable.tsx

```diff
@@ -27,14 +27,19 @@ import { VAULT_SECRET_PATH_ANNOTATION } from '../../constants';
 
 export const vaultSecretPath = (entity: Entity) => {
   const secretPath =
-    entity.metadata.annotations?.[VAULT_SECRET_PATH_ANNOTATION] ?? '';
+    entity.metadata.annotations?.[VAULT_SECRET_PATH_ANNOTATION];
 
   return { secretPath };
 };
 
 export const EntityVaultTable = ({ entity }: { entity: Entity }) => {
   const vaultApi = useApi(vaultApiRef);
   const { secretPath } = vaultSecretPath(entity);
+  if (!secretPath) {
+    throw Error(
```

Comment:
I assume the risk is low that someone hit this one as the card verifies that the annotation exists right?

Reply:
That's right. I couldn't really make that error to be thrown, as this is checked before by the annotation. Even though, I think is better to keep it, just in case something weird happen. Or what do you think?

Reply:
Thanks for the clarification. Let's keep it 👍